### PR TITLE
ZOOseo and ZOOtrack FAQ

### PR DESCRIPTION
**Pull request by dguzenko**

### CODE REVIEW

File: 02.extensions/zooseo/01.basics/06.faq/docs.md [hunk 1]
```diff
@@ -4,4 +4,10 @@ taxonomy:
...
 
-* **Is ZOOseo compatible with other Joomla! SEO extensions?** <br /> Yes, ZOOseo extension works with any Joomla! SEO extensions. They are complementary and work together.
\ No newline at end of file
+* **Why doesn't metadata load on the Item page?** - Please make sure that ZOOseo plugin is enabled and setup, and "ZOOseo Metadata Item" layout is configured.
+
+* **Is ZOOseo compatible with other Joomla! SEO extensions?** - Yes, ZOOseo extension works with any Joomla! SEO extensions. They are complementary and work together.
+
+* **Is it possible to set Open Graph meta information for a ZOO Category page?** - No, currently Open Graph meta tags can be applied only to ZOO Item pages.
+
+* **Does ZOOseo solve the problem with duplicated content?** - No, it is not a scope of ZOOseo plugin. To solve this problem ZOO sets the canoninal URL for an item. ZOOseo only prevents duplicating content by redirecting from URL with segments `/item` or `/category`to a new one (without segments).
```
Comment (ameli90) [inline 02.extensions/zooseo/01.basics/06.faq/docs.md]: space after /category`

Comment (dguzenko) [inline 02.extensions/zooseo/01.basics/06.faq/docs.md]: Done

File: 02.extensions/zooseo/01.basics/06.faq/docs.md [hunk 2]
```diff
@@ -4,4 +4,10 @@ taxonomy:
     category: docs
 ---
 
-* **Is ZOOseo compatible with other Joomla! SEO extensions?** <br /> Yes, ZOOseo extension works with any Joomla! SEO extensions. They are complementary and work together.
\ No newline at end of file
+* **Why doesn't metadata load on the Item page?** - Please make sure that ZOOseo plugin is enabled and setup, and "ZOOseo Metadata Item" layout is configured.
```
Comment (miljan-aleksic) [inline 02.extensions/zooseo/01.basics/06.faq/docs.md]: Don't use the (") carachter, use instead bold, cursive or code (**, *, `). If ZOOseo is enabled, already is set, so change the text to: "Please make sure that ZOOseo plugin is enabled and the *ZOOseo Metadata Item* layout is populated."

Comment (dguzenko) [inline 02.extensions/zooseo/01.basics/06.faq/docs.md]: Done

**Pull request merged**